source/_integrations/yolink.markdown

```diff
@@ -54,3 +54,4 @@ The integration is tested and verified for the following devices from YoLink:
 * YS4002/3 (Thermostat)
 * YS4906/8 YS7706 (Garage Door Sensor/ Garage Door Controller)
 * YS5707 (Dimmer)
+* YS6801/2 (MultiOutlet)
```

Comment:
This PR is missing a description, so I have to ask: Why is this added?

It helps if you take a second just to motivate your PRs.

Thanks 👍 

../Frenck

Reply:
Please create a core PR before creating documentation. You now have an orphan documentation PR, triggering the bot and my review...

Reply:
Sorry, I shoud do that before create this PR.